index.js

```diff
@@ -68,7 +83,18 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies,id) {
+  if(!id){
+    throw null
+  }
+  if(movies !== movies.length){
+    throw "Empty array."
+  }
+  return movies.find(movie => movie.imdb === id)
```

Comment:
Make sure you understand why you're failing tests like this. In this case, you shoudl be able to see from the terminal running your tests that you're finding the array empty every time. They're not _all_ empty, so something must be wrong with your condition, that that is always evaluating to `true`.

This and the other one are an example of leaving free points on the table. You had how to check if an array was empty earlier!